libs/feature/search/src/lib/favorites/favorite-star/favorite-star.component.spec.ts

```diff
@@ -150,19 +152,30 @@ describe('FavoriteStarComponent', () => {
           ])
           expect(favoritesService.addToFavorites).not.toHaveBeenCalled()
         })
+        beforeEach(() => {
+          component.record = {
+            ...RECORDS_SUMMARY_FIXTURE[0],
+            favoriteCount: 42,
+          }
+          ;(favoritesService as any).myFavoritesUuid$.next(['aaa'])
+          starToggle.newValue.emit(false)
+          fixture.detectChanges()
+        })
         it('decrease record favorite count by one', () => {
           expect(favoriteCountEl.textContent).toEqual(
             (component.record.favoriteCount - 1).toString()
           )
         })
+        it('removes the record from myFavoritesUuid$', async () => {
+          console.log(favoritesService.myFavoritesUuid$)
```

Comment:
remove
you should be able to run the test in your IDE instead of relying on logs.